.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 repos:
   - repo: https://www.github.com/nbQA-dev/nbQA
-    rev: 0.3.2
+    rev: 0.4.0
     hooks:
       - id: nbqa-black
+        additional_dependencies: [black==20.8b1]
```

Comment:
Would black be downloaded as part of this requirement? Or do we need to add it to the `requirements.txt` file?

Reply:
if you want to run this check via the command-line, then it'll need to be in `requirements.txt` (or `requirements-dev.txt`, I guess?)

If you just run it via pre-commit, e.g.
```
pre-commit run nbqa-black --all-files
```
or
```
pre-commit run nbqa-black --files <file_1> <file_2> ... <file-n>
```
then there's no need to add it to `requirements.txt`

----

> Nice, great to see progress, looking forward to the new features.

Thanks! `--nbqa-diff` flag (which lets you preview changes without applying them) is in progress

Reply:
Cool, I have included it in my PR #54 mostly I'll be using it but make it easy for another developer as well